.gitattributes

```diff
@@ -0,0 +1,94 @@
+# source code
```

Comment:
Can you please explain why is this file necessary here? What does it change? Do we expect PHP or XML files in this repo? It seems like a boilerplate for the sake of boilerplate, without any reason or effect.